### ISSUE TITLE
Unable to initialize Cerberus

### ISSUE THREAD

**Issue by Drewclam** [MTCKC/ProconXInput#13]
On Version 0.1.0-Alpha2 64-bit Release Build getting this error: "Unable to initialize Cerberus.
Error: Error connecting to the Cerberus service
Continuing, may not find the controller if it's hidden by HidGuardian.
Unable to find controller."

Also tried running HidCerberus.Srv.exe and then ProconXInput.exe. Says "Initialized HidCerberus. Unable to find controller."

Procon was working before, not sure why it isn't now.

**Comment by MTCKC** [comment]
Procon was working before as in ProconXInput or just connecting it to the PC and using it as a DInput device?

**Comment by Drewclam** [comment]
ProconXInput was working before.

**Comment by TroubledBubble** [comment]
I have the same problem but mine was NOT working before. I get the same results and everything, but I just installed this program and it hasn't worked. I have a wired only Switch Pro controller

**Comment by MTCKC** [comment]
Is the Pro Controller an official one? Currently the program only supports the original Switch Pro Controller, and it selects it in the code via it's hardware ID. The problem might be that the controller isn't picked up because it has a different hardware ID. If you have the know-how and can get me the hardware ID using Device Manager, I can whip up a version of ProconXInput that might work with it.

**Comment by TroubledBubble** [comment]
Ah, you're right! It's a PowerA switch controller... because I'm not savvy enough, I found the model number, 1503256-01, and what I THINK is the ID, L070912D0401, with zeros, not ohs. Thanks for helping me out!

**Comment by Drewclam** [comment]
Nope! My pro controller is an official one, if no one else is getting this error I'll just delete everything and try installing again when I have time

**Comment by Floplip** [comment]
Hi, I would love to use your program because I have a 3rd party switch controller and in games like fifa17 i can only run in a 45 degree angle or in a straight line. I found the following device instant path: USB\VID_045E&PID_028E\0B21065
That should mean that the ID is 045E 028E right?
It would be great if you could add it so I can use it :)

**Comment by SetoSenco** [comment]
I think I found a fix. I was having the same problem,  so I went ahead and accessed the Task Manager. Opened the More Details section, went to the services tab, and went to find the corresponding HIDCerberus.srv. I right clicked and clicked Run, ran ProconXinput and it worked. I suggest doing the same process and check

**Comment by SetoSenco** [comment]
Also-For other problems, you may need to turn off Windows Driver Signature enforcement
To do this:
1. Click and hold shift and restart you computer

2.Navigate to Trouble Shoot, Advanced Options, then Startup settings

3. Click Restart as prompted

4.Click F7 on your keyboard, and driver signature enforcement will be turned off *temporarily*. If errors occur after a few days, then repeat the steps again

**Comment by finckt** [comment]
I had the same problem, I uninstalled the drivers then installed back and it worked.

**Comment by AetherCollective** [comment]
Also, for those also having this issue, turn off secure boot in your BIOS if Window's Driver Signature enforcement refuses to disable.

**Comment by spmkd** [comment]
I have original Pro Controller and it was working previously.

Trying to run it after several months I got the same error.

For me, the issue was resolved by running the **uninstall Drivers.bat** file from the installation folder and then **install Drivers.bat**  again.

After this the the controller connected properly again.

By the way, **thanks a LOT** for making this app, it is great!!!
